romancal/lib/wcsinfo_to_wcs.py

```diff
@@ -29,22 +29,17 @@ def wcsinfo_to_wcs(
     wcs : WCS
         The WCS object created.
     """
-    pixelshift = models.Shift(
-        -wcsinfo.get("x0_projection", wcsinfo.get("x_ref", None)),
-        name="crpix1",
-    ) & models.Shift(
-        -wcsinfo.get("y0_projection", wcsinfo.get("y_ref", None)),
-        name="crpix2",
-    )
-    pixelscale = models.Scale(wcsinfo["pixel_scale"], name="scale") & models.Scale(
-        wcsinfo["pixel_scale"], name="scale"
-    )
+    crpix = [
+        wcsinfo.get("x0_projection", wcsinfo.get("x_ref", 0)),
+        wcsinfo.get("y0_projection", wcsinfo.get("y_ref", 0)),
+    ]
+    crval = [
+        wcsinfo.get("ra_projection_center", wcsinfo.get("ra_ref", 0)),
+        wcsinfo.get("dec_projection_center", wcsinfo.get("dec_ref", 0)),
+    ]
+    cdelt = [wcsinfo.get("pixel_scale", 1), wcsinfo.get("pixel_scale", 1)]
+
```

Comment:
FWIW, in resample.l3_to_wcs below, it's important to me to try to get a wcsinfo even if we can't find the right transform parameters.  But here I am totally okay erroring out if wcsinfo doesn't have what we need.  I think I made up the original x0_projection / ra_projection_center terminology in my original skycell draft a long time ago and it's not in rad, so the sane thing to do would probably be to retire the projection_center and ref quantities and just have this be wcsinfo['x_ref'], wcsinfo['ra_ref'].  But of course that's a separate issue.